dataprep/eda/correlation/compute.py

```diff
@@ -280,6 +331,60 @@ def correlation_nxn(
     return cordx, cordy, corrs
 
 
+def phik_correlation_nxn(
+    df: pd.DataFrame, columns: Optional[Sequence[str]] = None
+) -> Tuple[da.Array, da.Array, Dict[CorrelationMethod, da.Array], List[str]]:
+    """
+    Calcualation of a n x n phik correlation matrix of n columns
+    """
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore")
+        # Phi_k does not filter non-numerical with high cardinality
+        selcols = []
+        intcols = []
+        for col in df.columns.tolist():
+            try:
+                tmp = (
+                    df[col]
+                    .value_counts(dropna=False)
+                    .reset_index()
+                    .dropna()
+                    .set_index("index")
+                    .iloc[:, 0]
+                )
+                if tmp.index.inferred_type == "mixed":
+                    continue
+
+                if pd.api.types.is_numeric_dtype(df[col]):
+                    intcols.append(col)
+                    selcols.append(col)
```

Comment:
what does the 'selcols' means?

Reply:
Not all columns will be used to calculate the phik correlation. Some columns would make the calculation slow. Thus, we are supposed to select some columns to calculate the correlation. 
I imitate pandas-profiling to select columns. The selcols means the columns that are selected.

Reply:
Could you make a comment about the meaning of intcols and selcols in the code?

Reply:
I have added comments.